csrankings-9.csv

```diff
@@ -1518,6 +1520,7 @@ Ying Zhu,Ontario Tech University,https://engineering.ontariotechu.ca/people/ecse
 Ying Zou,Queen's University,https://www.ece.queensu.ca/people/Y-Zou,pCYlknMAAAAJ
 Ying-Dar Lin,National Chiao Tung University,http://speed.cis.nctu.edu.tw/~ydlin,6ZzMxhsAAAAJ
 Ying-Ping Chen,National Chiao Tung University,http://www.cs.nctu.edu.tw/~ypchen,5Fb9380AAAAJ
+Ying-Qing Xu,Tsinghua University,http://thfl.tsinghua.edu.cn/info/detailed/428,zSiDAt4AAAAJ

```

Comment:
Not visibly in CS, same issue as others.

Reply:
He is in information design school and is doing Graphics and HCI. He can advise students with a previous background in Computer Science. Not so sure if that is eligible.

Reply:
If all of them are not eligible then I guess I will close this Pull-Request. Sorry for the inconvenience, thank you!